src/server/zset_family.cc

```diff
@@ -650,6 +650,8 @@ double Aggregate(double v1, double v2, AggType atype) {
       return max(v1, v2);
     case AggType::MIN:
       return min(v1, v2);
+    case AggType::NOOP:
```

Comment:
Do we really need NOOP? I do not think that it's so computationally expensive to do SUM for example that it justifies a separate case.

Reply:
Since this is a switch-case there is no runtime cost for adding NOOP when using the normal aggregate types, while there is a runtime cost for doing SUM then not needed. 
IMO its a nice small costless speedup